@types/index.d.ts

```diff
@@ -1,19 +1,16 @@
 /// <reference types="node" />
+/// <reference lib="dom" />
```

Comment:
I've seen several issues when using both `node` and `dom` at the same time due to conflicting declarations. What was the reasoning for importing `URL`, `URLSearchParams`, and `Blob` from `dom` rather than `url` and `fetch-blob`?

Maybe the issues have been fixed in recent versions of the `node` and `dom` definitions though 🤔

Reply:
Um, I changed fetch-blob to be 100% spec compatible by introducing whatwg streams
i though you could just use the DOM blob cuz they are now indistinguishable

sry, but i'm no expert at TypeScript. I only use jsdoc for typing

Reply:
Is it possible to use a namespace like `NodeJS.Buffer.Blob` or something like it?

Reply:
Hmm, I'm not sure about this one actually... Maybe try it like this in the beta.10 and then we can change it if we encounter any issues?